src/Java.Interop.Tools.JavaTypeSystem/JavaModels/IJavaResolvable.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+using System.Collections.Generic;
+
+namespace Java.Interop.Tools.JavaTypeSystem.Models
+{
+	public interface IJavaResolvable
```

Comment:
Should probably be named `IJavaValidatable`, `IJavaVerifiable`, etc.  Lets try to avoid confusion around "Resolve", which is frequently used to mean *something else* in Cecil.